drivers/at86rf215/at86rf215.c

```diff
@@ -147,6 +147,10 @@ if (!IS_ACTIVE(CONFIG_AT86RF215_USE_CLOCK_OUTPUT)){
         reg |= AMCS_AACK_MASK;
     }
 
+    if (IS_USED(MODULE_AT86RF215_TIMESTAMP)) {
+        at86rf215_reg_write(dev, dev->BBC->RG_CNTC,
+                                 CNTC_EN_MASK | CNTC_CAPRXS_MASK);
```

Comment:
do we need to set this again when we wake from sleep?

Reply:
yes, all registers are reset after sleep

Reply:
Ah `at86rf215_set_rx_from_idle()` already takes care of that and calls `at86rf215_reset()`